cpu/stm32wb/Makefile.include

```diff
@@ -0,0 +1,16 @@
+CPU_ARCH = cortex-m4
+CPU_FAM  = stm32wb
+
+# "The Vector table must be naturally aligned to a power of two whose alignment
+# value is greater than or equal to number of Exceptions supported x 4"
+# CPU_IRQ_NUMOFF for stm32l4 boards is < 91+16 so (107*4 bytes = 428 bytes ~= 0x200)
+# RIOTBOOT_HDR_LEN can be set to 0x200
+RIOTBOOT_HDR_LEN ?= 0x200
+
+# Slot size is determined by "((total_flash_size - RIOTBOOT_LEN) / 2)".
+# If RIOTBOOT_LEN uses an uneven number of flashpages, the remainder of the
```

Comment:
```suggestion
# If RIOTBOOT_LEN uses an odd number of flashpages, the remainder of the
```
might be present in other places. Feel free to ignore if you don't want to change this.